static/js/public.js

```diff
@@ -13,6 +13,10 @@ require("./public/site.js");
 // jquery components
 require("./public/components/raty.js");
 
+// mailchimp requirements
+require("ajaxchimp");
+require('imports?this=>window!wowjs');
```

Comment:
I haven't either, to be honest I just copied this from https://github.com/matthieua/WOW/issues/233#issuecomment-173640648